app/controllers/api/users.php

```diff
@@ -1077,7 +1077,7 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $queueForEvents
             ->setParam('userId', $user->getId());
 
-        $response->dynamic(new Document($prefs), Response::MODEL_PREFERENCES);
+            $response->dynamic($user, Response::MODEL_USER);
```

Comment:
The response.model label needs to be updated up top too

Reply:
ok